### PR TITLE
adding hello endpoint

### PR DESCRIPTION
**Pull request by tompais**

### CODE REVIEW

File: src/main/kotlin/com/example/spring_api/controllers/HelloController.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.example.spring_api.controllers
+
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class HelloController {
+    @GetMapping("/hello")
+    fun hello() = "hello"
```
Comment (tompais) [inline src/main/kotlin/com/example/spring_api/controllers/HelloController.kt:9]: lalalaa

Comment (tompais) [inline src/main/kotlin/com/example/spring_api/controllers/HelloController.kt:9]: lalalala

File: src/test/kotlin/com/example/spring_api/unit/controllers/HelloControllerTest.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import assertk.assertions.isEqualTo
+import com.example.spring_api.controllers.HelloController
+import org.junit.jupiter.api.Test
+
+class HelloControllerTest {
+    private val helloController = HelloController()
+
+    @Test
+    fun helloTest() {
+        assertThat(helloController.hello()).isEqualTo("hello")
```
Comment (tompais) [inline src/test/kotlin/com/example/spring_api/unit/controllers/HelloControllerTest.kt:13]: lalalala

**Review by tompais — COMMENTED** [review]
comentarios generales

**Review by ezeallio — APPROVED** [review]

**Pull request merged**